src/controllers/notification.controller.js

```diff
@@ -0,0 +1,44 @@
+import NotificationService from '../services/notification.service';
+import UserService from '../services/user.service';
+import response from '../helpers/response.helper';
+
+/**
+ * This class contains functions for all notification controller.
+ * @class NotificationController
+ */
+class NotificationController {
+  /**
+   * Get notification for a user
+   * @param {Object} req The request object
+   * @param {Object} res The response object
+   * @returns {Promise} res
+   */
+  static async userNotifications(req, res) {
```

Comment:
I think we should name this method after the functionality that it provides for example _getUserNotification_

Reply:
Your right on that, thanks.